minecraft_servers/adventuresmp/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+    "server_name": "adventuresmp",
+    "nice_name": "AdventureSMP",
+    "direct_ip": "adventuresmp.de",
+    "supported_languages": ["de"],
+    "social": {
+        "web_shop": "https://shop.adventuresmp.de",
```

Comment:
Not reachable

Reply:
It's waiting approval, will be reachable in next 48 hours.